### ISSUE TITLE
Project 191 Missing 2019 Tofino Detections

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#516]
_Moved here from email discussion._

### TL;DR

 - There were 62 tags deployed May 1-7, 2019 near Tofino, BC, but only 16 of these tags were detected by receivers in the area. 
 - Only 1 of the 33 tags deployed near receiver PRNPR were ever detected
 - Both active receivers in the area detected 14 tags
 - An additional 2 tags were detected, but are not present in the dataset for project 191 because: 
  - **Tag 33938** was only ever detected immediately before deployment (12 hits)
  - **Tag 28932** is ambiguous with **[tag 22783](https://motus.org/data/tagDetections?id=22783)**, but detections are only allocated to the latter tag.

What to do:
 - [x] **Jeremy:** Rerun receiver deployments 4796 and 4795 in case metadata were not entered prior to data processing.
 - [ ] **Lucas:** Investigate whether receiver has been installed correctly
 - [ ] **Lucas:** Determine whether deployment times are correct for _all_ tags
 - [ ] **Catherine:** After Jeremy reruns receivers, sort out why those ambiguous detections aren't showing up in project 191.

---

### Details

Of the **62 tags** deployed around Tofino during the first week of May this year, only **16 tags** were detected on any of the receivers in the area.
![project191_Tofino_deployed_tags_2019](https://user-images.githubusercontent.com/35931796/60592870-e3cc5900-9d77-11e9-9b9e-c8dc59a1ffab.png)

The middle receiver was inactive at the time ([Dinner Island](https://motus.org/data/receiverDeployment?id=4794)), but the two others were operational and spaced 20 km apart:
![image](https://user-images.githubusercontent.com/35931796/60592390-eda18c80-9d76-11e9-9817-ea83852d5c0e.png)

Tags were deployed between the two operational receivers:
![tag_deploymentsMap_Tofino](https://user-images.githubusercontent.com/35931796/60592683-8a642a00-9d77-11e9-9a61-59eaac404c08.png)

But only **1 of the 33 tags** deployed in the east ([PRNPR](https://motus.org/data/receiverDeployment?id=4795)) were ever detected by a receiver, whereas **15 of 29 tags** deployed in the west ([Wickaninnish Inn](https://motus.org/data/receiverDeployment?id=4796)) were detected.
![project191_map_of_detections_near_Tofino_2019](https://user-images.githubusercontent.com/35931796/60592141-5fc5a180-9d76-11e9-8b09-a738c1df5481.png)

**Comment by Hussell** [comment]
Rerunning created a lot of new detections. Looking at the tag deployments, it looks like a lot of them were created after the receiver data was uploaded.

**Comment by leberrigan** [comment]
Yup, that did it. Thanks Jeremy. Looks like the ambig detections were also added to this project.

I think we're good here!
